internal/output/disk/disk.go

```diff
@@ -1,25 +1,27 @@
 package disk
 
 import (
+	"github.com/pkg/errors"
```

Comment:
why external `github.com/pkg/errors` not stdlib `errors` ? 
weird it is working since it wasn't added to go.mod